nncf/quantization/algorithms/accuracy_control/openvino_backend.py

```diff
@@ -26,10 +27,32 @@
 from nncf.openvino.graph.metatypes.openvino_metatypes import OVConcatMetatype
 from nncf.openvino.graph.metatypes.openvino_metatypes import OVOpMetatype
 from nncf.openvino.graph.model_utils import get_start_nodes_for_activation_path_tracing
+from nncf.openvino.graph.model_utils import model_has_state
 from nncf.openvino.graph.node_utils import get_bias_value
 from nncf.openvino.graph.node_utils import get_weight_value
 from nncf.openvino.graph.node_utils import is_node_with_bias
 from nncf.quantization.algorithms.accuracy_control.backend import AccuracyControlAlgoBackend
+from nncf.quantization.algorithms.accuracy_control.backend import PreparedModel
+
+
+class OVPreparedModel(PreparedModel):
+    """
+    Implementation of the `PreparedModel` for OpenVINO backend.
+    """
+
+    def __init__(self, model: ov.Model):
+        self.stateful = model_has_state(model)
+        self.compiled_model = ov.compile_model(model)
+        self.engine = None
+
+    @property
+    def model_for_inference(self) -> ov.CompiledModel:
+        return self.compiled_model
+
+    def get_engine(self) -> OVCompiledModelEngine:
```

Comment:
Maybe it also makes sense to decorate this method as a property

Reply:
Make sense